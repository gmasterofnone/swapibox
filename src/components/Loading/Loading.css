.loading {
  display: flex;
  justify-content: center;
}


.load {
  margin-top: 200px;
}