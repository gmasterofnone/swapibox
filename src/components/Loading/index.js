import React from 'react';

import './Loading.css';
import spin from '../Utils/assets/loading.svg'


const Loading = () => {
  return (
    <div className='loading'>
    <img className='load' src={spin} alt='loading'/> 
    </div>
  )
}

export default Loading;