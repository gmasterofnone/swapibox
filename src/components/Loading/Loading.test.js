/* eslint-disable */

import React from 'react';
import { shallow } from 'enzyme';
import Loading from '.';

describe('Loading', () => {
  
  it('matches the snapshot', () => {
    let wrapper = shallow(<Loading />);

    expect(wrapper).toMatchSnapshot();
  });

});