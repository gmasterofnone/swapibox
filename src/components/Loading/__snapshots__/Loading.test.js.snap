// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Loading />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <img
        alt="loading"
        className="load"
        src="loading.svg"
      />,
      "className": "loading",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "alt": "loading",
        "className": "load",
        "src": "loading.svg",
      },
      "ref": null,
      "rendered": null,
      "type": "img",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <img
          alt="loading"
          className="load"
          src="loading.svg"
        />,
        "className": "loading",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "loading",
          "className": "load",
          "src": "loading.svg",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
